Scaffold docs for LinkedIn Audiences (#3755)
segmentio/segment-docs: src/connections/destinations/catalog/actions-linkedin-audiences/index.md

11. Navigate back to **Engage > Audiences** and click on the Audience from Step 1. Click **Add Destinations** and select the LinkedIn Audiences destination you just created. In the settings that appear in the side panel, toggle the **Send Track** option on and disable **Send Identify**. Click **Save Settings**. 

![engage audience destination settings](images/image1.png)

The setup is complete and the Audience will start syncing to LinkedIn. Segment will automatically create a new DMP Segment in LinkedIn and add or remove users accordingly. The Audience will appear in your [LinkedIn Campaign Manager](https://www.linkedin.com/campaignmanager){:target="_blank"}, account under **Plan > Audiences > Matched**.

To sync additional Audiences from your Engage space, create a separate mapping in the LinkedIn Audiences destination. Navigate to **Connections > Destinations**, search and select the LinkedIn Audiences destination, and follow Steps 9-11 above.

{% include components/actions-fields.html settings="true"%}

## Troubleshooting

### Access & Refresh Tokens

> LinkedIn's OAuth access tokens have a time to live (TTL) of 60 days; refresh tokens have a TTL of one year. Segment will automatically update your access token as long as your refresh token is valid. You will not see any errors or interruptions in data delivery if your access token expires while your refresh token is valid.

Reviewer: ```suggestion
LinkedIn's OAuth access tokens have a time to live (TTL) of 60 days; refresh tokens have a TTL of one year. Segment automatically updates your access token as long as your refresh token is valid. You won't see any errors or interruptions in data delivery if your access token expires while your refresh token is valid.
```

> Upon expiry or revocation of a refresh token, you will see `Refresh Token Expired` errors in the Delivery Issues section of your LinkedIn Audiences destination **Event Delivery** tab.

Reviewer: ```suggestion
Upon expiry or revocation of a refresh token, you'll see `Refresh Token Expired` errors in the Delivery Issues section of your LinkedIn Audiences destination **Event Delivery** tab.
```
